### DISCUSSION TITLE
Problem saving Cover Art / Thumbnails from Library / Lastest page

### DISCUSSION THREAD

**Discussion by Sagnasty** [rern/rAudio#1149] [Bug Reports]
Running rAudio 20221210 with Update branch

To reproduce problem: 

- Update library (new content)
- Go to Library / Latest page
- Click on new album entry

Save icon is greyed out:
![image](https://user-images.githubusercontent.com/76575081/208463997-8883b93b-cd8f-4564-b168-038c0e6746da.png)

OR you may see this list of choices (not sure what causes the differences)
![image](https://user-images.githubusercontent.com/76575081/208464479-25b6c583-eaf0-4a8e-83c7-1f30c659661d.png)

Click on "Update Thumbnails" and the following message is display:
![image](https://user-images.githubusercontent.com/76575081/208465511-d6df5507-9443-465c-b77e-e8de087fa70c.png)

However, the cover art files are successfully saved to the root dir of the album despite the error message.

The behavior a few updates ago would have been to simply click on the save icon.   Very useful when including cover art for new content (as opposed to updating all cover art for large libraries).

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

**Comment by kw-sudo** [comment]
Something that has bothered me from the beginning is that when adding a new album, a thumbnail is never automatically created. I always have to do it manually (no matter which version). The user rights and privileges are correct.

Or am I wrong?

Oliver

**Reply by Sagnasty** [reply]
That would depend what permission levels the admin decided on when the storage was defined based on the specific use case.  

Some users do not care if their storage structure has full privileges and others decide that read only privileges is all that should be allowed.    

When defining permission levels, there is a obviously a big difference between a local USB drive and a NAS storage device which may have the port externally exposed - each case is different.    

Therefore, If all storage structures do not have full privileges, then it would not make sense to have an automated mechanism to write the thumbnails (or at least an automated mechanism that you cannot turn off).

**Reply by kw-sudo** [reply]
For this purpose I have a special 'audio user' that has full access (read/write) only to the music folder. To create thumbnail for new additions, I have to press the red marked button, then everything runs all by itself. Just not automatically when adding.

![thumbnail](https://github.com/rern/rAudio/assets/104136223/dd6fc3dd-3578-4790-8cfb-d914da67f361)

**Comment by Sagnasty** [comment]
Issue remains with release 20231022 

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

Jim
